hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RocksDatabase.java

```diff
@@ -289,10 +280,6 @@ public ColumnFamilyHandle getHandle() {
       return handle;
```

Comment:
The `@VisibleForTesting` tag was added by HDDS-8122 since it changed the method from `protected` to `public` for a unit test.    The `getHandle()` method is already used many times by in `RocksDatabase` and a few times in `RDBStore`.